scripts/sign_encrypt.py

```diff
@@ -58,11 +62,19 @@ def get_args(logger):
             command aliases:
               The command \'digest\' can be aliased by ''' + dat + '''
               The command \'stitch\' can be aliased by ''' + sat + '\n' + '''
-            example offline signing command using OpenSSL:
+            example offline signing command using OpenSSL for algorithm
+            TEE_ALG_RSASSA_PKCS1_PSS_MGF1_SHA256:
               base64 -d <UUID>.dig | \\
               openssl pkeyutl -sign -inkey <KEYFILE>.pem \\
-                  -pkeyopt digest:sha256 \\
-                  -pkeyopt rsa_padding_mode:pkcs1 | \\
+                  -pkeyopt digest:sha256 -pkeyopt rsa_padding_mode:pss \\
+                  -pkeyopt rsa_pss_saltlen:digest \\
+                  -pkeyopt rsa_mgf1_md:sha256 |\\
```

Comment:
minor: space char before `\\` for consistency.
Same at line 77.

Reply:
Fix it